create_db.py

```diff
@@ -16,10 +16,8 @@ def create_default_user(email, password):
         ask_email = True
         while ask_email:
             email = input("Enter email for super_admin    : ")
-            if not re.match(r'[^@]+@[^@]+\.[^@]+', email):
+            if not re.match(r'^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w+$', email):
```

Comment:
It is checking only "@" and "." rather it should check alphanumeric characters also, to bring more accuracy in checking the email ids.